lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -375,6 +393,33 @@ class LighthouseReportViewer {
       self.opener.postMessage({opened: true}, '*');
     }
   }
+
+  /**
+   * @param {PSIParams} params
+   */
+  _loadFromPSI(params) {
+    const loadingOverlayEl = document.createElement('div');
+    loadingOverlayEl.classList.add('lh-loading-overlay');
+    loadingOverlayEl.textContent = 'Waiting for Lighthouse results ...';
+    find('.viewer-placeholder-inner', document.body).classList.add('lh-loading');
+    document.body.appendChild(loadingOverlayEl);
+    this._psi.fetchPSI(params).then(response => {
+      if (!response.lighthouseResult) {
+        if (response.error) {
+          // eslint-disable-next-line no-console
+          console.error(response.error);
+          loadingOverlayEl.innerText = response.error.message;
```

Comment:
also should use `logger.error`

Reply:
done

Reply:
I just noticed that error message flashes in a bottom left corner. Since that exists, maybe the loading overlay should be removed on an error?